src/full/Agda/TypeChecking/Serialise/Instances/Errors.hs

```diff
@@ -96,7 +96,7 @@ instance EmbPrj Warning where
       valu [16]         = valuN WithoutKFlagPrimEraseEquality
       valu [17, a]      = valuN InstanceWithExplicitArg a
       valu [18, a, b]   = valuN InfectiveImport a b
-      valu [19, a, b]   = valuN InfectiveImport a b
+      valu [19, a, b]   = valuN CoInfectiveImport a b
```

Comment:
Oops, thanks for spotting that! Don't you need to update [currentInterfaceVersion](https://github.com/agda/agda/blob/master/src/full/Agda/TypeChecking/Serialise.hs#L62-L63) as well?

Reply:
I don't think so: the decoding has changed but the encoding is still the same.

Reply:
Good point.